docs/alerts/monitors/alert-response.md

```diff
@@ -229,18 +229,39 @@ A _Monitor_ creates an _Alert_. Using the options below, you're subscribing to a
 :::
 
 
-#### From your Alerts list
+### From your Alerts list
 * Right-click on a row item > click **Subscribe**
 * Hover your mouse over a row, click the three-dot kebab menu > select **Subscribe**
 * Single-click on a row item > on the opened Alert page, click the three-dot kebab menu > **Subscribe to Monitor**
 
 
-#### From your Monitors list
+### From your Monitors list
 * Right-click on a row item > click **Subscribe**
 * Hover your mouse over a row > click the three-dot kebab menu > click **Subscribe**
 * Single-click on a row item > in the side panel (Monitor Details), click **More Actions** > **Subscribe**
 
 
+## Muting Schedules
+
+:::note
+Make sure you have the **Viewing and Manage Muting Schedules** permissions.
```

Comment:
We should indicate that you need "Manage Muting Schedules" + "Manage Monitors" capabilities to define/edit a muting schedule in fine-grained permissions are not enabled for your account. If they are, you would need "Manage Muting Schedules" + "Admin Monitors" capabilities.

Reply:
@tarunk2 I don't see any of these mentioned in our [master list of permissions](https://help.sumologic.com/docs/manage/users-roles/roles/role-capabilities/). Should those be added and defined there?

Reply:
Yes. cc @angadrandhawa1 @kevin-sumo

Reply:
Yes, we need to add "View Muting Schedules" and "Manage Muting Schedules" there. Other ones are already mentioned under https://help.sumologic.com/docs/manage/users-roles/roles/role-capabilities/#alerts

Reply:
Done - let me know what you think